docs/reference/commandline/dockerd.md

```diff
@@ -740,6 +740,19 @@ to add multiple lower directory support for OverlayFS. This option should
 only be used after verifying this support exists in the kernel. Applying
 this option on a kernel without this support will cause failures on mount.
 
+##### `overlay2.size`
+
+Sets the default size of the container. It is supported only when the
```

Comment:
`Sets the default size of the container` does not sound right.
Maybe `Sets the default max size for containers`